modules/inflowwind/src/InflowWind_Driver.f90

```diff
@@ -109,73 +108,16 @@ PROGRAM InflowWind_Driver
-   CLSettingsFlags%IfWIptFile          =  .FALSE.        ! InflowWind input filename given as command line argument
-   CLSettingsFlags%Summary             =  .FALSE.        ! create a summary at command line? (data extents in the wind file)
-   CLSettingsFlags%SummaryFile         =  .FALSE.        ! create a summary file of the output?
-   CLSettingsFlags%TStart              =  .FALSE.        ! specified time to start at
-   CLSettingsFlags%NumTimeSteps        =  .FALSE.        ! specified a number of timesteps
-   CLSettingsFlags%NumTimeStepsDefault =  .FALSE.        ! specified 'DEFAULT' for number of timesteps
-   CLSettingsFlags%DT                  =  .FALSE.        ! specified a resolution in time
-   CLSettingsFlags%DTDefault           =  .FALSE.        ! specified 'DEFAULT' for resolution in time
-   CLSettingsFlags%FFTcalc             =  .FALSE.        ! do an FFT
-   CLSettingsFlags%WindGrid            =  .FALSE.        ! Requested output of wind data on a grid -- input file option only
-   CLSettingsFlags%XRange              =  .FALSE.        ! specified a range of x      -- command line option only -- stored as GridCtrCoord and GridDelta
-   CLSettingsFlags%YRange              =  .FALSE.        ! specified a range of y      -- command line option only -- stored as GridCtrCoord and GridDelta
-   CLSettingsFlags%ZRange              =  .FALSE.        ! specified a range of z      -- command line option only -- stored as GridCtrCoord and GridDelta
-   CLSettingsFlags%Dx                  =  .FALSE.        ! specified a resolution in x -- command line option only, 0.0 otherwise
-   CLSettingsFlags%Dy                  =  .FALSE.        ! speficied a resolution in y
-   CLSettingsFlags%Dz                  =  .FALSE.        ! specified a resolution in z
-   CLSettingsFlags%PointsFile          =  .FALSE.        ! points filename to read in  -- command line option only
-   CLSettingsFlags%WindGridOutputInit  =  .FALSE.        ! Wind Grid output file not started
-   CLSettingsFlags%FFTOutputInit       =  .FALSE.        ! FFT output file not started
-   CLSettingsFlags%PointsOutputInit    =  .FALSE.        ! Points output file not started
-   CLSettingsFlags%Verbose             =  .FALSE.        ! Turn on verbose error reporting?
-   CLSettingsFlags%VVerbose            =  .FALSE.        ! Turn on very verbose error reporting?
-   CLSettingsFlags%WrHAWC              =  .FALSE.        ! don't convert to HAWC format
-   CLSettingsFlags%WrBladed            =  .FALSE.        ! don't convert to Bladed format
-   CLSettingsFlags%WrVTK               =  .FALSE.        ! don't convert to VTK format
-   CLSettingsFlags%WrUniform           =  .FALSE.        ! don't convert to UniformWind format
-
-      ! Initialize the driver settings to their default values (same as the CL -- command line -- values)
-   Settings       =  CLSettings
-   SettingsFlags  =  CLSettingsFlags
-
+    CLSettings%ProgInfo = ProgInfo       
```

Comment:
I'm not sure I understand where setting the defaults for the `CLSettings` are now (maybe some aren't needed).